howto/deploy-k8s-prod/README.md

```diff
@@ -0,0 +1,187 @@
+
+1. Upgrading a CLI version (optional but recommended)
+2. Updating the control plane
+3. Updating the data plane (sidecars)
+
+### 1. Upgrading the CLI
+
+To upgrade the Dapr CLI, [download a release version](https://github.com/dapr/cli/releases) of the CLI that matches the Dapr runtime version.
+For example, if upgrading to Dapr 0.9.0, download a CLI version of 0.9.x.
+
+After you downloaded the binary, it's recommended you put the CLI binary in your path.
+
+### Updating the control plane
+
+#### Saving the current certificates
+
+When upgrading to a new version of Dapr, it is recommended we carry over our Root and Issuer certificates instead of re-generating them, which might cause a downtime for applications that make use of service invocation or actors.
+
+To get the current certificates, run the following command:
+
+```
+kubectl get secret dapr-trust-bundle -o yaml -n dapr-system
+
+apiVersion: v1
+data:
+  ca.crt: <ROOT-CERTIFICATE-VALUE>
+  issuer.crt: <ISSUER-CERTIFICATE-VALUE>
+  issuer.key: <ISSUER-KEY-VALUE>
+kind: Secret
+```
+
+Copy the contents of `ca.crt`, `issuer.crt` and `issuer.key` and base64 decode them.
+
+Finally, you should have the following files saved on your disk:, containing the base64 decoded text from the secret:
+
+1. ca.crt
+2. issuer.crt
+3. issuer.key
+
+#### Updating the control plane pods
+
+Next, we need to find a Helm Chart version that installs our desired version of Dapr and perform a `helm upgrade` operation.
+
+First, update the Helm Chart repos:
+
+```bash
+helm repo update
+```
+
+List all charts in the Dapr repo:
+
+```bash
+helm search repo dapr
+
+NAME     	CHART VERSION	APP VERSION	DESCRIPTION
+dapr/dapr	0.4.3        	0.9.0      	A Helm chart for Dapr on Kubernetes
+dapr/dapr	0.4.2        	0.8.0      	A Helm chart for Dapr on Kubernetes
+```
+
+The APP VERSION column tells us which Dapr runtime version is installed by the chart.
+
+Use the following command to upgrade Dapr to your desired version:
+
+```bash
+helm upgrade dapr dapr/dapr --version <CHART-VERSION> --namespace dapr-system --reset-values --set-file dapr_sentry.tls.root.certPEM=ca.crt --set-file dapr_sentry.tls.issuer.certPEM=issuer.crt --set-file dapr_sentry.tls.issuer.keyPEM=issuer.key
+```
+
+Kubernetes will now issue a rolling update.
+Wait until all the new pods appear as running:
+
+```bash
+kubectl get po -n dapr-system -w
+
+NAME                                     READY   STATUS    RESTARTS   AGE
+dapr-dashboard-dcd7cc8fb-ml2p8           1/1     Running   0          12s
+dapr-operator-7b57d884cd-6spcq           1/1     Running   0          12s
+dapr-placement-86b76f6545-vkc6f          1/1     Running   0          12s
+dapr-sentry-7c7bf75d8b-dcnhm             1/1     Running   0          12s
+dapr-sidecar-injector-7b847db96f-kwst9   1/1     Running   0          12s
+```
+
+You can verify the health and version of the control plane using the Dapr CLI:
+
+```bash
+dapr status -k
```

Comment:
This CLI should also show the state of the dashboard pod if running

Reply:
Yeah, but that's an issue that needs to be open on the CLI.